### PR TITLE
1055 change the name of my institution lal ijclab and add me in author list

### PR DESCRIPTION
**Pull request by sylvielsstfr**
I have kept the old name LAL for those author still affiliated to LAL in the list.

### CODE REVIEW

**Comment by timj** [comment]
To clarify, the institution did not change formally from LAL to IJCLAB for everyone, but LAL still exists as an entity that people are still affiliated to?

**Comment by sylvielsstfr** [comment]
Normaly every ex LAL people are in IJCLab with a few exceptions for those that changed to another institute. I have not checked for them. I will go through the authorlist to check this and correct it.

**Comment by sylvielsstfr** [comment]
I have run yamlint on the file and remove the trailing blank.

File: etc/authordb.yaml [hunk 1]
```diff
@@ -639,14 +639,6 @@ authors:
     initials: Rahul
     name: Biswas
     orcid: null
-  blancg:
```
Comment (timj) [inline etc/authordb.yaml:642]: Can you comment as to why this author was removed from the file?

**Comment by timj** [comment]
I rebased your branch to remove the merge from main.

File: etc/authordb.yaml (hunk 1 above)
Comment (sylvielsstfr) [reply etc/authordb.yaml:642]: Yes he left LSST

**Pull request merged**